src/h5web/visualizations/heatmap/LogScaleToggler.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { FiEyeOff, FiEye } from 'react-icons/fi';
+import styles from './HeatmapVis.module.css';
+
+interface Props {
+  logScale: boolean;
+  setLogScale: React.Dispatch<React.SetStateAction<boolean>>;
```

Comment:
Avoid using this type in props, as it's too specific -- if we need to pass a function that does something with the boolean before setting the state, then we'll have to change the type here too.

Reply:
What do you suggest instead then ?

Reply:
Ah sorry, just `(val: boolean) => void`